tests/testthat/test-tidyeval.R

```diff
@@ -0,0 +1,11 @@
+context("tidyeval")
+
+test_that("bang bang", {
+  expect_warning(test_collection("tidyeval", "bang_bang",
+                                transformer = style_text), NA)
+})
+
+test_that("bang bang", {
```

Comment:
Anything different works ;-) If the test descriptions are identical for two tests, it's slightly harder to find the source of a test failure.